### ISSUE TITLE
Reported comments

### ISSUE THREAD

**Issue by jagramonte70** [rthackston-gsu/discussion#33]
If someone's comment is reported by a user, the site should allow override for the user to add their own comment, disregarding the un-critiqued comment that has been reported by that user.

**Comment by rthackston-gsu** [comment]
It sounds like you're asking for the ability for a user to provide a new comment when someone has reported their original comment to the admins. The original commenter won't actually know their comment has been reported.

Did you mean something different?

**Comment by jagramonte70** [comment]
Yes, I did mean something different. I attempted to clarify in a direct email to you.

**Comment by rthackston-gsu** [comment]
Clarification: Allow students to skip critiquing a comment if they "report" the comment for some reason.
